displaylink-sleep.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2015 DisplayLink (UK) Ltd.
```

Comment:
Where is that file from? Is there any document explaining the messages you're sending to those sockets?